Jaeger tracing fails in native image

Following exception is thrown when using native image together with Jaeger tracing:
```
--------------------------
2022.04.05 16:17:44:819 SEVERE io.helidon.webserver.RequestRouting Thread[nioEventLoopGroup-4-2,10,main]: Unexpected error occurred during routing!
com.oracle.svm.core.jdk.UnsupportedFeatureError: Runtime reflection is not supported for public static java.time.Instant java.time.Instant.now()
```

This can be fixed by adding the following `reflect-config.json` to `META-INF/native-image/{maven-group-id}/{maven-artifact-id}`:
```
[
    {
        "name": "java.time.Instant",
        "methods": [
            {
                "name": "now",
                "parameterTypes": []
            }
        ]
    }
]
```

I think the fix might have been incomplete as I can reproduce a failure with Helidon SE 2.5.1. Details:

* Helidon 2.5.1
* SE
*  GraalVM 21.3.0 Java 11 CE (Java Version 11.0.13+7-jvmci-21.3-b05)

## To Reproduce

Generate project using Helidon SE 2.5.1 quickstart.

Add the following to `pom.xml`:

```
<dependency>
    <groupId>io.helidon.tracing</groupId>
    <artifactId>helidon-tracing</artifactId>
</dependency>
<dependency>
    <groupId>io.helidon.tracing</groupId>
    <artifactId>helidon-tracing-jaeger</artifactId>
</dependency>
```

Add to `application.yaml`:
```
tracing:
  service: helidon-se-1
```

Update `Main`:

```
import io.helidon.tracing.TracerBuilder;
...
WebServer server = WebServer.builder(createRouting(config))
                .config(config.get("server"))
                .tracer(TracerBuilder.create(config.get("tracing")).build())
                .addMediaSupport(JsonpSupport.create())
                .build();
```

Build with native image and run. Fails with:

```
2022.07.06 08:37:03 INFO io.helidon.common.LogConfig Thread[main,5,main]: Logging at runtime configured using classpath: /logging.properties
Exception in thread "main" java.lang.ExceptionInInitializerError
    at io.jaegertracing.internal.clock.SystemClock.<clinit>(SystemClock.java:47)
    at io.jaegertracing.internal.JaegerTracer$Builder.<init>(JaegerTracer.java:553)
    at io.jaegertracing.internal.JaegerTracer$Builder.<init>(JaegerTracer.java:564)
    at io.jaegertracing.Configuration.createTracerBuilder(Configuration.java:245)
    at io.jaegertracing.Configuration.getTracerBuilder(Configuration.java:232)
    at io.helidon.tracing.jaeger.JaegerTracerBuilder.build(JaegerTracerBuilder.java:456)
    at io.helidon.examples.quickstart.se.Main.startServer(Main.java:48)
    at io.helidon.examples.quickstart.se.Main.main(Main.java:31)
Caused by: java.lang.IllegalStateException: Could not setup microseconds accurate time supplier
    at io.jaegertracing.internal.clock.MicrosAccurateClock.<clinit>(MicrosAccurateClock.java:64)
    ... 8 more
Caused by: java.lang.NoSuchFieldException: EPOCH
    at java.lang.Class.getField(DynamicHub.java:1139)
    at io.jaegertracing.internal.clock.MicrosAccurateClock.<clinit>(MicrosAccurateClock.java:56)
    ... 8 more
```

## Possible Fix

Updating 
`tracing/jaeger/src/main/resources/META-INF/native-image/io.helidon.tracing/helidon-tracing-jaeger/reflect-config.json`
with the following appears to resolve  the issue (note, I did not attempt to trim this down to the smallest change):

```
 {
        "name": "java.time.Instant",
        "allPublicConstructors": true,
        "allPublicMethods": true,
        "allPublicFields": true,
        "methods": [
            {
                "name": "now",
                "parameterTypes": []
            }
        ]
    },
    {
        "name": "java.time.temporal.ChronoUnit",
        "allPublicConstructors": true,
        "allPublicMethods": true,
        "allPublicFields": true
    },
    {
        "name": "java.time.temporal.Temporal",
        "allPublicConstructors": true,
        "allPublicMethods": true,
        "allPublicFields": true
    }
```

--- a/tracing/jaeger/src/main/resources/META-INF/native-image/io.helidon.tracing/helidon-tracing-jaeger/reflect-config.json
+++ b/tracing/jaeger/src/main/resources/META-INF/native-image/io.helidon.tracing/helidon-tracing-jaeger/reflect-config.json
@@ -103,11 +103,26 @@
     },
     {
         "name": "java.time.Instant",
+        "allPublicConstructors": true,
+        "allPublicMethods": true,
+        "allPublicFields": true,
         "methods": [
             {
                 "name": "now",
                 "parameterTypes": []
             }
         ]
+    },
+    {
+        "name": "java.time.temporal.ChronoUnit",
+        "allPublicConstructors": true,
+        "allPublicMethods": true,
+        "allPublicFields": true
+    },
+    {
+        "name": "java.time.temporal.Temporal",
+        "allPublicConstructors": true,
+        "allPublicMethods": true,
+        "allPublicFields": true
     }
 ]

